coding/coding_tests/multilang_utf8_string_test.cpp

```diff
@@ -102,3 +102,15 @@ UNIT_TEST(MultilangString_Unique)
   TEST(s.GetString(1, cmp), ());
   TEST_EQUAL(cmp, "yyy", ());
 }
+
+UNIT_TEST(MultilangString_LangNames)
+{
+  TEST_EQUAL("Беларуская", StringUtf8Multilang::GetLangNameByCode(StringUtf8Multilang::GetLangIndex("be")), ());
+
+  auto const & langs = StringUtf8Multilang::GetSupportedLanguages();
+  // Using size_t workaround, because our logging/testing macroses do not support passing POD types by value,
+  // only by reference. And passing constexpr by reference converts it from compile-time constant to runtime constant.
```

Comment:
Please, clarify how it's possible that constexpr will lose it's compile-time-constness and become a runtime constant.